### PR TITLE
Add libraries required for CM_EMU

### PR DESCRIPTION
**Pull request by tfzhu**

### CODE REVIEW

File: devops/scripts/install_build_tools.sh [hunk 1]
```diff
@@ -12,7 +12,13 @@ apt update && apt install -yqq \
       python3-pip \
       zstd \
       ocl-icd-libopencl1 \
-      vim
+      vim \
+      libffi-dev \
+      libva-dev \
+      libtool \
+      autoconf \
+      pkg-config
```
Comment (bader) [inline devops/scripts/install_build_tools.sh:20]: > Add libraries required for CM_EMU

Why do we need autoconf and pkg-config? Do you suggest build CM_EMU from sources? Can't we just download pre-build package?

Comment (tfzhu) [reply devops/scripts/install_build_tools.sh]: Removed. If anything missing, @dongkyunahn-intel please add in this file.

Comment (bader) [reply devops/scripts/install_build_tools.sh]: Thanks regarding the rest of the packages - they have -dev suffix, which I suppose will install sources for these libs. I don't think we need their sources. I suppose cm emulator is dynamically linked to some libraries and these should be available on the system. I don't expect we need to build them from sources.
@dongkyunahn-intel, could you clarify what exactly is missing in standard Ubuntu image?

Comment (alexbatashev) [reply devops/scripts/install_build_tools.sh]: > they have -dev suffix, which I suppose will install sources for these libs. I don't think we need their sources.

IIRC, `-dev` suffix installs include files, which may be needed to compile the code.

Comment (tfzhu) [reply devops/scripts/install_build_tools.sh]: Used -dev package for other containers. This should be the same.

Comment (bader) [reply devops/scripts/install_build_tools.sh]: > Used -dev package for other containers. This should be the same.

Could you clarify what do you mean by that, please?
From my POV, for other back-ends we build and link with a "loader" library and rest of the dependencies are supplied in binary form, so we don't need `-dev` packages for Level Zero or OpenCL drivers.
I expect CM emulator backend to "be the same". The library we link with downloaded by CMake, so the rest of the packages are not used in the build. Is it different for CM emulator?

It's not clear if really need `-dev` packages for CM emulator backend or not.

Comment (tfzhu) [reply devops/scripts/install_build_tools.sh]: Needs @dongkyunahn-intel 's input. For the log of building on RHEL, it do need -dev package.

Comment (dongkyunahn-intel) [reply devops/scripts/install_build_tools.sh]: Until CM_EMU can be provided as pre-built library package, it has to be built online. CMakefile for ESIMD_EMULATOR support contains this online building step (https://github.com/intel/llvm/blob/c87806381b309faea185dd5f461ff4d72cf60d08/sycl/plugins/esimd_emulator/CMakeLists.txt#L65), and building step requires `-dev` packages for certain header files.

Once prebuilt CM_EMU library packages becomes available, we won't need `-dev` packages.

**Review by dongkyunahn-intel — APPROVED** [review]

File: devops/scripts/install_build_tools.sh (hunk 1 above)
Comment (bader) [reply devops/scripts/install_build_tools.sh]: > Once prebuilt CM_EMU library packages becomes available

@dongkyunahn-intel , do you know when they become available? Can we wait for that?

Comment (dongkyunahn-intel) [reply devops/scripts/install_build_tools.sh]: It seems prebuilt CM_EMU library package is generated internally. I'm asking my colleagues how to get and use it.

Comment (dongkyunahn-intel) [reply devops/scripts/install_build_tools.sh]: I was told that internal release failed in public release process due to new regression. I don't think we can wait until next internal release & public release process.

Comment (bader) [reply devops/scripts/install_build_tools.sh]: :(

**Review by bader — APPROVED** [review]

**Review by alexbatashev — APPROVED** [review]

**Pull request merged**